google_assistant/README.md

```diff
@@ -106,4 +106,17 @@ Finally, run the push to talk sample. The sample records a voice query after a k
 ```
 python -m pushtotalk
 ```
-`
+# 6) Troubleshooting
+This helps us do a simple speaker and microphone test, just to make sure that the hardware is functional. 
+```
+$ speaker-test -t wav
+$ arecord --format=S16_LE --duration=5 --rate=16000 --file-type=raw out.raw
+$ aplay --format=S16_LE --rate=16000 --file-type=raw out.raw
+```
+If there is an error that says a certain package is missing, check the path of the imported packages in the code and make sure they are in the right place. 
```

Comment:
It'd be nice to add a note about changing the default audio device for debugging.

See, hw: parameter for arecord and aplay